test/response_test.rb

```diff
@@ -393,8 +353,8 @@ class RubySamlTest < Minitest::Test
     describe '#xpath_first_from_signed_assertion' do
       it 'not allow arbitrary code execution' do
         malicious_response_document = fixture('response_eval', false)
-        response = OneLogin::RubySaml::Response.new(malicious_response_document)
-        response.send(:xpath_first_from_signed_assertion)
+        malicious_response = OneLogin::RubySaml::Response.new(malicious_response_document)
+        malicious_response.send(:xpath_first_from_signed_assertion)
         assert_equal($evalled, nil)
```

Comment:
is it possible we can redo this spec case using a different approach? ( I personally don't like using global vars )